metadata.json

```diff
@@ -1,39 +1,70 @@
 {
   "name": "MiamiOH-shibboleth_idp",
-  "version": "1.3.0",
+  "version": "1.4.0",
   "author": "Dirk Tepe",
   "summary": "Manage Shibboleth IdP",
-  "license": "GPL-3.0+",
+  "license": "GPL-3.0",
   "source": "https://github.com/MiamiOH/puppet-shibboleth_idp.git",
   "project_page": "https://github.com/MiamiOH/puppet-shibboleth_idp",
   "issues_url": "https://github.com/MiamiOH/puppet-shibboleth_idp/issues",
-  "tags": ["shibboleth", "shib", "idp"],
   "dependencies": [
-    {"name": "puppetlabs/stdlib", "version_requirement": ">= 4.2.0 < 5.0.0"},
-    {"name": "puppetlabs/concat", "version_requirement": ">= 2.2.1 < 5.0.0"},
-    {"name": "puppet/archive", "version_requirement": ">= 0.5.0 < 2.0.0"},
-    {"name": "camptocamp/openssl", "version_requirement": ">= 1.8.2 < 2.0.0"},
-    {"name": "unibet/profiled", "version_requirement": ">= 0.1.4 < 1.0.0"},
-    {"name": "camptocamp/systemd", "version_requirement": ">= 0.4.0 < 3.0.0"}
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.2.0 < 5.0.0"
+    },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 2.2.1 < 5.0.0"
+    },
+    {
+      "name": "puppet/archive",
+      "version_requirement": ">= 1.2.0 < 6.0.0"
+    },
+    {
+      "name": "camptocamp/openssl",
+      "version_requirement": ">= 1.8.2 < 2.0.0"
+    },
+    {
+      "name": "unibet/profiled",
+      "version_requirement": ">= 0.1.4 < 1.0.0"
+    },
+    {
+      "name": "camptocamp/systemd",
+      "version_requirement": ">= 0.4.0 < 3.0.0"
+    }
   ],
   "operatingsystem_support": [
     {
       "operatingsystem": "RedHat",
-      "operatingsystemrelease": ["7"]
+      "operatingsystemrelease": [
+        "7"
+      ]
     },
     {
       "operatingsystem": "CentOS",
-      "operatingsystemrelease": ["7"]
+      "operatingsystemrelease": [
+        "7"
+      ]
     },
     {
       "operatingsystem": "OracleLinux",
-      "operatingsystemrelease": ["7"]
+      "operatingsystemrelease": [
+        "7"
+      ]
     }
   ],
   "requirements": [
     {
       "name": "puppet",
       "version_requirement": ">= 3.7.0 < 5.0.0"
     }
-  ]
+  ],
+  "tags": [
+    "shibboleth",
+    "shib",
+    "idp"
+  ],
+  "pdk-version": "1.17.0",
+  "template-url": "https://github.com/puppetlabs/pdk-templates#1.9.0",
+  "template-ref": "tags/1.9.0-0-g7281db5"
```

Comment:
didn't we find out that 1.9.0 tested puppet 5 and 6 in one of travis or gitlab ci?

Did we find a new version that tested puppet 4 and 5 in both?

Reply:
1.9.1 does 5/6, 1.9.0 is what seems to work